design/peer-awareness.md

```diff
@@ -0,0 +1,64 @@
+# Peer Awareness
+
+Target version: Yorkie 0.1.2
```

Comment:
I am not sure if it is absolutely necessary to specify the version. Can you explain why you entered the version?

Reply:
I thought writing a release version would allow us to think about the scope of features and focus more on our goals when writing your design documentation.